src/control/cmd/agent/main.go

```diff
@@ -127,6 +131,13 @@ func agentMain(log *logging.LeveledLogger, opts *cliOptions) error {
 	ctx, shutdown := context.WithCancel(context.Background())
 	defer shutdown()
 
+	if opts.ConfigPath == "" {
+		defaultConfigPath := path.Join(configDir, defaultConfigFile)
+		if _, err := os.Stat(defaultConfigPath); err == nil {
```

Comment:
Since this logic to update the config path is repeated for the agent, server, and dmg, would be nice to extract it as a common helper function.